nanomq/webhook_inproc.c

```diff
@@ -573,36 +576,36 @@ webhook_thr(void *arg)
 
 	nng_dialer_start(dialer, NNG_FLAG_NONBLOCK);
 
-	for (i = 0; i < conf->web_hook.pool_size; i++) {
+	for (i = 0; i < works_num; i++) {
 		works[i] = alloc_work(sock, &conf->web_hook, &conf->exchange);
 		works[i]->id = i;
 		works[i]->mqtt_sock = &mqtt_sock;
 	}
 	// NanoMQ core thread talks to others via INPROC
-	if ((rv = nng_listen(sock, WEB_HOOK_INPROC_URL, NULL, 0)) != 0) {
-		log_error("webhook nng_listen %d", rv);
+	if ((rv = nng_listen(sock, HOOK_IPC_URL, NULL, 0)) != 0) {
+		log_error("hook nng_listen %d", rv);
 		return;
 	}
 
-	for (i = 0; i < conf->web_hook.pool_size; i++) {
+	for (i = 0; i < works_num; i++) {
```

Comment:
sure this num works right?

Reply:
It depends on the web_hook.pool_size and number of exchanges

Reply:
Of course right